GoogleDataLogger/GoogleDataLogger/Classes/GDLLogEvent.h

```diff
@@ -44,6 +44,9 @@ typedef NS_ENUM(NSInteger, GDLLogQoS) {
 /** The log map identifier, to allow backends to map the extension property to a proto. */
 @property(readonly, nonatomic) NSString *logMapID;
 
+/** The identifier for the backend this log will eventually be sent to. */
+@property(readonly, nonatomic) NSInteger logTarget;
```

Comment:
Why is an integer needed in addition to the logMapID?

Reply:
the logMapID is for the backend to use to map a given log extension to the appropriate proto.
the logTarget is an identifier, probably defined by a future shared proto, that defines the backend this log should be sent to.